org.openkj.OpenKJ.yml

```diff
@@ -0,0 +1,67 @@
+app-id: org.openkj.OpenKJ
+runtime: org.kde.Platform
+runtime-version: 5.15
+sdk: org.kde.Sdk
+command: openkj
+modules:
+  - codecs/liba52.json
+  - codecs/libmpeg2.json
+  - codecs/gst-plugins-ugly.json
+  - codecs/libass.json
+  - codecs/ffmpeg.json
+  - codecs/gst-libav.json
+  - codecs/gst-plugins-bad.json
+  - name: rubberband 
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://github.com/breakfastquay/rubberband.git
+        branch: default
+  - name: fmt 
+    buildsystem: cmake-ninja
+    builddir: yes
+    config-opts:
+      - "-DFMT_TEST=OFF"
+    sources:
+      - type: git
+        url: https://github.com/fmtlib/fmt.git
+        branch: master 
+  - name: spdlog 
+    buildsystem: cmake-ninja
+    builddir: yes
+    config-opts:
+      - "-DSPDLOG_BUILD_EXAMPLE=OFF"
+      - "-DSPDLOG_BUILD_TESTS=OFF"
+    sources:
+      - type: git
+        url: https://github.com/gabime/spdlog.git
+        branch: v1.x
+  - name: taglib 
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://github.com/taglib/taglib.git
+        branch: master 
+  - name: OpenKJ
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://github.com/openkj/openkj.git
+        tag: v2.0.4-release
+        commit: dd102391f4cd57728f215ef6e26b37794e569ffb 
+finish-args: 
```

Comment:
these are usually toward the top of the manifest

Reply:
Moved